src/blas/impl/KokkosBlas3_trsm_spec.hpp

```diff
@@ -130,7 +130,7 @@ struct TRSM {
     std::ostringstream os;
     os << "KokkosBlas::trsm currently supports only TPL interface."
        << "Please enable Host BLAS or cuBLAS."
-       << "KokkosKernels implmentation will be added in future release.";
+       << "KokkosKernels implementation will be added in future release.";
```

Comment:
I still wish there were a sequential fall-back implementation.  It's useful for testing whether the BLAS is correct.

Reply:
Thanks, though, for the spelling fix! :-)

Reply:
@mhoemmen I am willing to add a sequential implementation. It is just I do not have enough time for it now and we would like to have this PR in the 3.0 release :-). I sure work on it after merging.

Reply:
@vqd8a I'm just expressing a wish :-)